.circleci/config.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           key: v2-gradle-cache-{{ checksum "build.gradle.kts" }}
       - run:
           name: Run build test
-          command: ./gradlew clean build
+          command: ./gradlew clean build --daemon
```

Comment:
we don't use daemon mode in CI normally - is there a specific reason for this change?

Reply:
Add a comment in the code pls :)